vignettes/package_extensions.Rmd

```diff
@@ -0,0 +1,95 @@
+---
+title: "Package Extensions"
+output: 
+  rmarkdown::html_vignette
+    
+vignette: >
+  %\VignetteIndexEntry{Package Extensions}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+link <- function(text, url) {
+  return(
+    paste0(
+      "[", text, "]",
+      "(", url, ")"
+    )
+  )
+}
+
+dyn_link <- function(text,
+                     base_url,
+                     relative_url = "",
+                     # Change to TRUE when admiral adopts multiversion docs
+                     is_multiversion = TRUE,
+                     multiversion_default_ref = "main") {
+  url <- paste(base_url, relative_url, sep = "/")
+  if (is_multiversion) {
+    url <- paste(
+      base_url,
+      Sys.getenv("BRANCH_NAME", multiversion_default_ref),
+      relative_url,
+      sep = "/"
+    )
+  }
+  return(link(text, url))
+}
+
+# Other variables
+admiral_homepage <- "https://pharmaverse.github.io/admiral"
+admiraldev_homepage <- "https://pharmaverse.github.io/admiraldev"
+admiralci_homepage <- "https://pharmaverse.github.io/admiralci"
+```
+
+## Introduction
+
+`{admiral}` is made up of a family of packages and we foresee this only growing over time to cover more specific areas such as TA (Therapeutic Area) package extensions, with a wider range of companies and individuals getting on board to join development efforts. This step-by-step guidance talks through our recommendations on how new development teams would go about creating such package extensions. It is critical that this guidance is followed, as our users need to feel a consistent experience when working across `{admiral}` packages. If an `{admiral}` package extension doesn't follow these conventions then we wouldn't include it under pharmaverse and as part of the `{admiral}` family.
+
+## Step-by-step Guidance to Create a Package Extension
+_Note: The ordering numbers below are suggested but don't all need to strictly be followed in this sequence._
+
+1. Raise the need for a new `{admiral}` package extension with the sponsors (Ross Farrugia ross.farrugia@roche.com & Ben Arancibia benjamin.c.arancibia@gsk.com). The naming convention needs to be `{admiralxxx}` and we request that the scope is not targeted overly narrow, for example instead of a package extension for HIV we'd prefer one across virology. Otherwise the number of packages may become unmanageable.
```

Comment:
I think this might need to change to Dean Grundy.

Reply:
Yes sounds good, do you have an email please?

Reply:
On second thought - why not just asked them to reach out to the slack channel and remove the emails.  

Or ask them to reach out to the package maintainer - but not provide the name. 

Just trying to not have to update this section if roles change, people move on, etc.

Reply:
yes that works! any of us could then pick up such requests and follow up.
@bms63 please add Dean to our future sponsors/leads meetings if that gets confirmed. i spoke to him a while back and we worked together for years before at Roche so great to collaborate again if so

Reply:
Agree, changed.